types/react-resizable/react-resizable-tests.tsx

```diff
@@ -1,10 +1,7 @@
-import * as React from "react";
```

Comment:
Are all these code style changes a result of the DefinitelyTyped repo changing their rules, or your editor? Probably good to avoid needless style changes if possible

Reply:
Looks like the repo https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/.prettierrc.json#L4